client/main/options.js

```diff
@@ -2,166 +2,178 @@
 
 'use strict';
 
-const Options = function() {
+const SuperClass = require('../common/superclass');
+const _ = require('underscore');
 
-};
+const OptionType = {
```

Comment:
OptionType is an object that contains the 'Option, Group, Terms...' classes, its not a class itself. This is so that a OptionType.create(...) could be added which determines the appropriate option to be instantiated using OptionType[....] for the constructor.